src/quo2/components/tabs/segmented_tab.cljs

```diff
@@ -1,5 +1,5 @@
 (ns quo2.components.tabs.segmented-tab
-  (:require [quo2.components.tabs.tab :as tab]
+  (:require [quo2.components.tabs.tab.view :as tab]
```

Comment:
@ilmotta : This change seems to have broken the segmented control in latest develop 

This is how the segmented control looked like before :
![Screenshot 2023-02-03 at 2 28 50 PM](https://user-images.githubusercontent.com/64726664/216556583-5df32e96-e22a-4c06-8861-bb9ced4623a0.png)

This is how it looks like now : 
![Screenshot 2023-02-03 at 2 31 06 PM](https://user-images.githubusercontent.com/64726664/216557095-3138e860-10be-4e20-a6d6-1228db379d1d.png)

Reply:
Thank you @siddarthkay. I'll investigate an try to fix this.

Reply:
@ilmotta @siddarthkay hi! is this problem somehow visible in current UI? Just want to understand if I have missed it during testing.

Reply:
@pavloburykh : no, this component is hardly used anywhere and I only found it while working on a new screen.
I could see the broken in the sandbox that we have for quo2 preview :)

Reply:
This issue will definitely be fixed by mid next week @pavloburykh. It should be a quick one, but hopefully as Sid said it's not impacting anything, so lower priority.

Reply:
got it! thanx @ilmotta @siddarthkay